pom.xml

```diff
@@ -53,7 +53,7 @@
         <dep.tempto.version>175</dep.tempto.version>
         <dep.gcs.version>2.0.0</dep.gcs.version>
         <dep.errorprone.version>2.3.4</dep.errorprone.version>
-        <rubix.version>0.3.9</rubix.version>
+        <rubix.version>0.3.10</rubix.version>
```

Comment:
If I create another commit for this then the commit without this will not compile. We used some constants and methods from new release jar which were not there in the last one.

Reply:
You could make version update commit the first one.

Reply:
Done